container-images/ramalama-fedora/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+COPY ../scripts /scripts
```

Comment:
I know it's not obvious apologies, but "ramalama" means use a generic driver, aka the vulkan version of kompute. I think what we should do here is:

/scripts/build_llama_and_whisper.sh "rocm" "fedora"

with the first parameter being the GPU/AI driver (rocm in this case) and the second parameter could be fedora, ubi, etc.

Reply:
SGTM